package.json

```diff
@@ -137,7 +137,7 @@
     "all": true
   },
   "engines": {
-    "node": ">=8.3.0"
+    "node": ">=10.16.3"
```

Comment:
Does it specifically need to be greater than 10.16.3? Do earlier 10 versions not suffice?

Reply:
Ah yes, probably should be `>=10.0.0`